SWLOR.Game.Server/Service/ItemService.cs

```diff
@@ -319,37 +326,76 @@ public string OnModuleExamine(string existingDescription, NWPlayer examiner, NWO
             {
                 description += _color.Orange("FP Regen Bonus: ") + examinedItem.FPRegenBonus + "\n";
             }
+            if (examinedItem.PilotingBonus > 0)
+            {
+                description += _color.Orange("Piloting Bonus: ") + examinedItem.PilotingBonus + "\n";
+            }
             if (examinedItem.BaseAttackBonus > 0)
             {
-                description += _color.Orange("Base Attack Bonus: ") + examinedItem.BaseAttackBonus + "\n";
+                if (WeaponBaseItemTypes.Contains(examinedItem.BaseItemType))
+                {
+                    description += _color.Orange("Base Attack Bonus: ") + examinedItem.BaseAttackBonus + "\n";
+                }
+                else
+                {
+                    description += _color.Red("Base Attack Bonus (ignored due to item type): ") + examinedItem.BaseAttackBonus + "\n";
+                }
             }
             if (examinedItem.SneakAttackBonus > 0)
             {
                 description += _color.Orange("Sneak Attack Bonus: ") + examinedItem.SneakAttackBonus + "\n";
             }
             if (examinedItem.DamageBonus > 0)
             {
-                description += _color.Orange("Damage Bonus: ") + examinedItem.DamageBonus + "\n";
+                if (WeaponBaseItemTypes.Contains(examinedItem.BaseItemType))
+                {
+                    description += _color.Orange("Damage Bonus: ") + examinedItem.DamageBonus + "\n";
+                }
+                else
+                {
+                    description += _color.Red("Damage Bonus (ignored due to item type): ") + examinedItem.DamageBonus + "\n";
+                }
             }
             if (examinedItem.CustomItemType != CustomItemType.None)
             {
                 string itemTypeProper = string.Concat(examinedItem.CustomItemType.ToString().Select(x => char.IsUpper(x) ? " " + x : x.ToString())).TrimStart(' ');
                 description += _color.Orange("Item Type: ") + itemTypeProper + "\n";
             }
 
+            // Check for properties that can only be applied to limited things, and flag them here.
+            // Attack bonus, damage, base attack bonus: weapons only
+            // AC - armor items only.
+            ItemProperty ip = _.GetFirstItemProperty(examinedItem);
+            while (_.GetIsItemPropertyValid(ip) == TRUE)
+            {
+                if (_.GetItemPropertyType(ip) == (int) CustomItemPropertyType.ComponentBonus)
+                {
+                    switch (_.GetItemPropertySubType(ip))
+                    {
+                        case (int)ComponentBonusType.ACUp:
+                        {
+                            description += _color.Cyan("AC can only be applied to Shields, Armo(u)r and Helmets.  On other items, it will be ignored.\n");
```

Comment:
Lol. As much as I love the (u) we probably should stick to one consistent naming.